### PR TITLE
Update us-geological-survey.csl

### PR DESCRIPTION
**Pull request by jbellino-usgs**
1. Updated citation to alphabetically order citations.
2. Miscellaneous updates to bibliography and added webpage "elif" to better represent webpage information.

### CODE REVIEW

File: us-geological-survey.csl [hunk 1]
```diff
@@ -121,8 +150,8 @@
               <et-al term="and others"/>
               <label form="short" prefix=" "/>
             </names>
-            <text variable="container-title" prefix=" " suffix=","/>
-            <text variable="collection-title" prefix=" " suffix=","/>
+            <text variable="event" prefix=" " suffix=" --"/>
```
Comment (adam3smith) [inline us-geological-survey.csl]: the style really wants two ugly hyphens there? not a pretty little en dash or so?

Comment (jbellino-usgs) [inline us-geological-survey.csl]: Hey Sebastian,

I don't recognize that slice of code or suffix...I focused mainly on the
sections related to report and webpage documents. I'm no longer in front of
my computer and won't be so for a week - I'll have a look when I get back.

On Friday, March 20, 2015, Sebastian Karcher <notifications@github.com>
wrote:

> In us-geological-survey.csl
> <https://github.com/citation-style-language/styles/pull/1463#discussion_r26870609>
> :
>
> > @@ -121,8 +150,8 @@
> >                <et-al term="and others"/>
> >                <label form="short" prefix=" "/>
> >              </names>
> > -            <text variable="container-title" prefix=" " suffix=","/>
> > -            <text variable="collection-title" prefix=" " suffix=","/>
> > +            <text variable="event" prefix=" " suffix=" --"/>
>
> the style really wants two ugly hyphens there? not a pretty little en dash
> or so?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/citation-style-language/styles/pull/1463/files#r26870609>
> .
>


-- 
Sent from Gmail Mobile

Jason C. Bellino
jbellino@usgs.gov
US Geological Survey
4446 Pet Ln
Lutz, FL 33559

Comment (rmzelle) [inline us-geological-survey.csl]: @jbellino-usgs, you can update this pull request yourself by clicking the pencil icon at https://github.com/jbellino-usgs/styles/blob/patch-1/us-geological-survey.csl. If you need an en-dash, one way is to just copy and paste it from the internet (e.g. from http://en.wikipedia.org/wiki/Dash, or directly from here: "–").

Comment (jbellino-usgs) [inline us-geological-survey.csl]: @rmzelle @adam3smith Hi guys, sorry about the confusion here - I am new to github and haven't used it very much yet so I'm woefully unfamiliar with pull requests and the like. I was also on mobile at the time the question of the en dash came up. I tried to update the pull request last week with the en dash, but it looks like I didn't do it correctly - I just updated the pull request and the Travis CI is running now.

**Comment by rmzelle** [comment]
Great, thanks! (it's not necessary to close and reopen the pull request, in case you did that intentionally)

**Comment by jbellino-usgs** [comment]
@rmzelle Actually I thought maybe I had unknowingly opened another parallel pull request last week  and so I closed it only to realize that it was, in fact, the original request. Thanks for your help here...

**Comment by rmzelle** [comment]
No, looking at https://github.com/jbellino-usgs?tab=activity it looks like you only had one :).

**Pull request merged**